cfscrape/__init__.py

```diff
@@ -56,6 +58,17 @@
 https://github.com/Anorov/cloudflare-scrape/issues\
 """
 
+class CaptchaProvokingCiphersRemover(HTTPAdapter):
+    url = "https://"
+
+    def init_poolmanager(self, *args, **kwargs):
+        context = create_urllib3_context()
+        problematic_ciphers = ("AES128-SHA",)
```

Comment:
I would think that the net effect would be the same as `DEFAULT_CIPHERS += ':!AES128-SHA'` but I haven't check it yet. If so, it would avoid having to recompute the list.

Reply:
Unfortunately, the `get_ciphers` method does not exist on SSLContext instances object on python2.
Also, I'm afraid that it breaks compatibility in some ways because in causes the following error when trying to connect to https://ssllabs.com/ssltest/viewMyClient.html

```
requests.exceptions.SSLError: HTTPSConnectionPool(host='ssllabs.com', port=443): Max retries exceeded with url: /ssltest/viewMyClient.html (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1056)')))
```

Reply:
Would you mind adjust the PR to make use of `DEFAULT_CIPHERS += ':!SHA1' ` for now? The `DEFAULT_CIPHERS` can be imported from `urllib3.util.ssl_`.

Reply:
The `proxy_manager_for` method should be overridden the same ways as `init_poolmanager` to fix this issue when using proxies as well: https://github.com/kennethreitz/requests/blob/master/requests/adapters.py#L166

Reply:
Thanks for all the feedback! Will integrate your proposal, and then have a look at python2. I've seen the SSLV3_ALERT_HANDSHAKE_FAILURE before with python2 in requests without cfscrape

Reply:
Interesting... Seems like I recall something similar having to do with openssl TLSv1.3 not being completely backported.